spec/services/gpo_reminder_sender_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+RSpec.describe GpoReminderSender do
+  describe '#send_emails' do
+    WAIT_BEFORE_SENDING_REMINDER = 14.days
+    TIME_DUE_FOR_REMINDER = Time.zone.now - WAIT_BEFORE_SENDING_REMINDER
+    TIME_NOT_YET_DUE = TIME_DUE_FOR_REMINDER + 1.day
+    TIME_YESTERDAY = Time.zone.now - 1.day
```

Comment:
this is an anti-patter IMO, these constants are being defined in the global namepace and can pollute other tests. I would recommend redoing them as `let`'s

```suggestion
    let(:wait_before_sending_reminder) { 14.days }
    let(:time_due_for_reminder) { Time.zone.now - wait_before_sending_reminder }
    let(:time_not_yet_due) { time_due_for_reminder + 1.day }
    let(:time_yesterday) { 1.day.ago }
```

Reply:
Changed.